src/main/java/com/google/devtools/build/lib/remote/RemoteModule.java

```diff
@@ -392,6 +392,19 @@ public void beforeCommand(CommandEnvironment env) throws AbruptExitException {
 
     if ((enableHttpCache || enableDiskCache) && !enableGrpcCache) {
       initHttpAndDiskCache(env, credentials, authAndTlsOptions, remoteOptions, digestUtil);
+      buildEventArtifactUploaderFactoryDelegate.init(
+        new ByteStreamBuildEventArtifactUploaderFactory(
+          executorService,
+          env.getReporter(),
+          verboseFailures,
+          actionContextProvider.getRemoteCache(),
+          remoteOptions.remoteInstanceName,
```

Comment:
This option only makes sense for a gRPC cache, so we should probably pass in an empty string instead.